airbyte-cdk/python/unit_tests/sources/mock_server_tests/test_resumable_full_refresh.py

```diff
@@ -157,6 +165,42 @@ def test_resumable_full_refresh_sync(self, http_mocker):
         assert actual_messages.state_messages[3].state.stream.stream_state == AirbyteStateBlob()
         assert actual_messages.state_messages[3].state.sourceStats.recordCount == 0.0
 
+    @HttpMocker()
+    def test_resumable_full_refresh_sync_with_configured_schema(self, http_mocker):
```

Comment:
nit: what is this test testing that isn't covered by other tests?

Reply:
Just the new option in the builder but I think I can do something better and update the original `_create_catalog` to stop using CatalogBuilder (will be deprecated) and go full with ConfiguredAirbyteStreamBuilder.